src/wheatley.ts

```diff
@@ -771,26 +768,39 @@ export class Wheatley {
         this.increment_message_counters(message);
     }
 
-    get_initial_wheatley_info(): wheatley_database_info {
-        return {
-            id: "main",
-            server_suggestions: {
-                last_scanned_timestamp: SERVER_SUGGESTION_TRACKER_START_TIME,
-            },
-            modmail_id_counter: 0,
-            the_button: {
-                button_presses: 0,
-                last_reset: Date.now(),
-                longest_time_without_reset: 0,
-            },
-            starboard: {
-                delete_emojis: [],
-                ignored_emojis: [],
-                negative_emojis: [],
-                repost_emojis: [],
-            },
-            moderation_case_number: 0,
-            watch_number: 0,
-        };
+    async migrate_db(database: WheatleyDatabase) {
+        const proxy = database.create_proxy<{
+            wheatley: mongo.Document;
+            component_state: mongo.Document;
+        }>();
+
+        const bot_singleton = await proxy.wheatley.findOne({ id: "main" });
+        if (bot_singleton) {
+            M.log("migrating database…");
```

Comment:
very minor:
```suggestion
            M.log("migrating database...");
```

Reply:
Can change, but I thought we're already using UTF-8 in other places so might as well :P